### PR TITLE
First public version of the Sysdig Helm package

### PR DESCRIPTION
**Pull request by bencer**

### CODE REVIEW

**Comment by bencer** [comment]
I signed it!

**Comment by lachie83** [comment]
@k8s-bot ok to test

**Comment by bencer** [comment]
@lachie83 the running pods test if expected to fail since requires an AccessKey as documente in NOTES.txt, should we add an exception?

File: stable/sysdig/templates/daemonset.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+         path: /boot
+      - name: modules-vol
+        hostPath:
+         path: /lib/modules
+      - name: usr-vol
+        hostPath:
+          path: /usr
+      hostNetwork: true
+      hostPID: true
+      containers:
```
Comment (lachie83) [inline stable/sysdig/templates/daemonset.yaml:39]: Looks like the resource constraints are missing. Would you mind adding them?

File: stable/sysdig/templates/secrets.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
```
Comment (lachie83) [inline stable/sysdig/templates/secrets.yaml:6]: Would you like to add a consistent label set to all resources created by this chart?

**Comment by bencer** [comment]
@lachie83 I've added some improvements based on your comments and a couple more. Thank you.

**Comment by lachie83** [comment]
@bencer Looks great!

**Pull request merged**